test/quantization/test_numeric_suite_fx.py

```diff
@@ -142,3 +144,96 @@ def compare_and_validate_results(float_model, q_model):
         q_model = convert_fx(prepared_model)
 
         compare_and_validate_results(backup_prepared_model, q_model)
+
+    @skipIfNoFBGEMM
+    def test_compare_model_stub_conv_static_fx(self):
+        r"""Compare the output of static quantized conv layer and its float shadow module"""
+
+        def calibrate(model, calib_data):
+            model.eval()
+            with torch.no_grad():
+                for inp in calib_data:
+                    model(*inp)
+
+        def compare_and_validate_results(float_model, q_model, module_swap_list, data):
+            ob_dict = compare_model_stub_fx(
+                float_model, q_model, module_swap_list, data
+            )
+            expected_ob_dict_keys = {"conv.stats"}
+            print(ob_dict.keys())
+            self.assertTrue(ob_dict.keys() == expected_ob_dict_keys)
+            self.assertEqual(len(ob_dict), 1)
+            for k, v in ob_dict.items():
+                self.assertTrue(len(v["float"]) == len(v["quantized"]))
+                for i, val in enumerate(v["quantized"]):
+                    self.assertTrue(v["float"][i].shape == v["quantized"][i].shape)
+
+        qengine = torch.backends.quantized.engine
+        qconfig = get_default_qconfig(qengine)
+        qconfig_dict = {"": qconfig}
+
+        model_list = [ConvModel(), ConvBnReLUModel()]
+
+        for float_model in model_list:
+            float_model.eval()
+
+            prepared_model = prepare_fx(float_model, qconfig_dict)
+
+            backup_prepared_model = copy.deepcopy(prepared_model)
+
+            # Run calibration
+            calibrate(prepared_model, self.img_data_2d)
+            q_model = convert_fx(prepared_model)
+
+            module_swap_list = [nn.Conv2d, nn.intrinsic.modules.fused.ConvReLU2d]
+            compare_and_validate_results(
+                backup_prepared_model,
+                q_model,
+                module_swap_list,
+                self.img_data_2d[0][0],
+            )
+
+    @skipIfNoFBGEMM
```

Comment:
Use override_qengines to be consistent?

Reply:
I changed to override_qeninges for all APIs in other PR . Also calibrate to test_only_eval  (#49266)